src/policy/fees.cpp

```diff
@@ -11,7 +11,9 @@
 #include "primitives/transaction.h"
 #include "streams.h"
 #include "txmempool.h"
+#include "unlimited.h"
 #include "util.h"
+#include "utilmoneystr.h"
```

Comment:
In Initialize(), lets DbgAssert if decay = 1 (set to a reasonable value in release mode).  this will prevent div-by-0 later on.

Reply:
decay is hard coded to the default value defined in fees.h but i don't see why this would hurt anything.

Reply:
i looked more at this and i think a runtime error would be more appropriate than a DbgAssert as having an invalid value here will mess up this entire section of code or cause a crash if decay == 1